chromium/background-scripts/update.js

```diff
@@ -271,12 +284,33 @@ function destroyTimer() {
   }
 }
 
+function clear_replacement_update_channels() {
+  let keys = [];
+  for (let update_channel of combined_update_channels) {
+    if(update_channel.replaces_default_rulesets){
+      util.log(util.NOTE, update_channel.name + ': You have a new version of the extension.  Clearing any stored rulesets, which replace the new extension-bundled ones.');
+      keys.push('rulesets-timestamp: ' + update_channel.name);
+      keys.push('rulesets-stored-timestamp: ' + update_channel.name);
+      keys.push('rulesets: ' + update_channel.name);
+    }
+  }
+
+  return new Promise(resolve => {
+    chrome.storage.local.remove(keys, resolve);
+  });
+}
+
 async function initialize(store_param, cb) {
   store = store_param;
   background_callback = cb;
 
   await loadUpdateChannelsKeys();
 
+  if (await store.local.get_promise('extensionTimestamp', 0) != extension_timestamp) {
```

Comment:
`!==` should be used to avoid edge cases with values that are loosely equal (e.g. `0` and empty string or `false`).